LeagueBreakoutWebPage.html

```diff
@@ -0,0 +1,46 @@
+      <h3>                    The reason for why I made this game is because as a child I was really fascinated of playing this game and level 2 gave me an opportunity to simulate a similar game to it. I also made this game due to the game always being a challenge which still is to this day.                 </h3>
+      <hr size='3'>
+      <h2>                    2. What I Learned                </h2>
+      <h3>                    Something I learned from user testing is that it never comes out right the first time. An example of this was the paddle movement speed, it was going too slow which made it difficult to hit the ball. Thanks to user testing feedback, I changed the paddle speed to accommodate to the majority.                </h3>
+      <hr size='3'>
+      <h2>                    3. Architecture of My Game                </h2>
+      <h3>                    The foundations of the game are the paddle, ball, and blocks. It starts off by the ball hitting the blocks and using its collision box to determine where it will go next. When a block is hit for the first time, it goes from its starting color which is red to green, if its hit twice the blocks color turns blue and if it his hit a third time, the block will be destroyed. The paddle is primarily used as a support for the ball since the ball can not go under or else it counts as a loss.                 </h3>
+      <hr size='3'>
+      <h2>                    4. Technical Implementation                </h2>
+      <h3>                    Most of the technical implementations happen in the game panel. It handles the timer which keeps my code updated 60 times per second and this is important because I want to know where the ball is all the time to determine steps to make. Other implements I have is the keyListener and this is used primarily for the user to be able to press the keys to make the paddle move either left or right. I also have an ActionListener that keeps track of when  we are changing states. In total there are 3 states, menu state, game state, and end state so the ActionListener is making sure to change screens whenever the enter key is not being typed.                  </h3>
+      <hr size='3'>
+      <h2>                    5. Most Challenging Part                </h2>
+      <h3>                    The most difficult part of this project was the physics involved. Checking whether the ball was hit by the sides was relatively easy but checking whether the ball was hit on the top or bottom became difficult. I had to place breakpoints to debug the problem and this helped me overcome the problem because I realized that it was a condition that messed up the code. Another difficulty was when the ball hit the corner, it would hit one corner and completely get rid of the block. A way to overcome this was to measure half of the balls width and if half of the ball was greater on one side then it will react to what I set for that side.                 </h3>
```

Comment:
>I realized that it was a condition that messed up the code

Can you be more specific about the conditions that caused the bugs?  Explain what the problem was and how you fixed it.